new/LIT6389MalkeaTewodros.xml

```diff
@@ -0,0 +1,75 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng"
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng"
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT6389MalkeaTewodros" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">መልክአ፡ ቴዎድሮስ፡</title>
```

Comment:
Glottal stop in transliteration?

Reply:
@DenisNosnitsin1970 I took this transliteration from the way it is done in most of the Chaine Repertoire, so: Malkǝʾa śǝʿǝl, Malkǝʾa ʾEdom or have I missed something else? Sorry to drag this one out so much!

Reply:
That is fine, I meant, you have transliteration "Malkǝʿa" with Ayn, Ethiopic with Alif.

Reply:
Aha! I should have spotted that!

Reply:
Aha! I should have spotted that!